filer/models/abstract.py

```diff
@@ -68,7 +68,8 @@ def file_data_changed(self, post_init=False):
                 self._width, self._height = PILImage.open(imgfile).size
                 imgfile.seek(0)
             except Exception:
-                self._width, self._height = None, None
+                if post_init is False:
```

Comment:
Could you explain the rationale behing this?
Probably it would be useful to add a comment. I remember setting these has been quite controversial in the past

Reply:
done